<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

  
    <title>Publications - Biodata Analysis Group</title>
  
  
  <meta name="description" content="The Biodata Analysis Group website">
  <meta name="author" content="Fotis Psomopoulos">

  <!-- CSS -->
  <!-- Bootstrap is included in main.css -->
  <link rel="stylesheet" href="/assets/theme/css/main.css">
  <link rel="stylesheet" href="/assets/theme/css/font-awesome.min.css">
  <link rel="stylesheet" href="/assets/theme/css/pygments.css">

  <!-- Fonts -->
  <link href="https://fonts.googleapis.com/css?family=Lora:400,400i,700,700i&display=swap" rel="stylesheet">

  <!-- Feeds -->
  <link rel="alternate" type="application/rss+xml" title="Biodata Analysis Group - RSS feed" href="https://biodataanalysisgroup.github.io/feed.xml" >
</head>


  <body>
    <nav class="navbar navbar-expand-lg navbar-light bg-light" id="main-nav">
  <div class="container">
    <a class="navbar-brand d-flex align-items-center" href="/">
      
        <img src="/assets/img/groupLogo/helix.png" class="mr-2">
      
      Biodata Analysis Group
    </a>
    
      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbar-menu">
        Menu <i class="fa fa-bars"></i>
      </button>
      <div class="collapse navbar-collapse" id="navbar-menu">
        <ul class="navbar-nav ml-auto">
        
          
            <li class="navbar-item">
              <a class="nav-link" href="/" >Home</a>
            </li>
          
        
          
            <li class="nav-item dropdown">
              <a class="nav-link dropdown-toggle" href="#" role="button" data-toggle="dropdown">
                The Group
              </a>
              <div class="dropdown-menu">
              
                
                  <a class="dropdown-item" href="/team/" >Team</a>
                
              
                
                  <a class="dropdown-item" href="/coc/" >Code of Conduct</a>
                
              
              </div>
          
        
          
            <li class="navbar-item">
              <a class="nav-link active" href="/publications/" >Publications</a>
            </li>
          
        
          
            <li class="nav-item dropdown">
              <a class="nav-link dropdown-toggle" href="#" role="button" data-toggle="dropdown">
                Research
              </a>
              <div class="dropdown-menu">
              
                
                  <a class="dropdown-item" href="/activities/" >Activities</a>
                
              
                
                  <a class="dropdown-item" href="/projects/" >Projects</a>
                
              
                
                  <a class="dropdown-item" href="/collaborations/" >Collaborations</a>
                
              
                
                  <div class="dropdown-divider"></div>
                
              
                
                  <h6 class="dropdown-header">External resources</h6>
                
              
                
                  <a class="dropdown-item" href="https://github.com/BiodataAnalysisGroup" target="_blank">Lab GitHub</a>
                
              
              </div>
          
        
          
            <li class="navbar-item">
              <a class="nav-link" href="/blog/" >News</a>
            </li>
          
        
          
            <li class="navbar-item">
              <a class="nav-link" href="/about/" >About</a>
            </li>
          
        
        </ul>
      </div>
    
  </div>
</nav>


    <main>
      
  <header class="jumbotron bg-no-image">

    <div class="container">
      <div class="row">
        <div class="col-md-10 col-lg-8 mx-auto">
          <h1>Publications</h1>
          
          <div class="lead">
</div>
          
          

          <div class="categories">
            
          </div>
        </div>
      </div>
    </div>
  </header>


      <div class="container content">
          
        <div class="row">
  <div class="col-md-10 col-lg-8 mx-auto">

    <!-- # Bibliography -->

<p><a href="#book"><strong>Book</strong></a>   <a href="#book-chapter"><strong>Book Chapter</strong></a>   <a href="#journals"><strong>Journals</strong></a>   <a href="#conferences-and-announcements"><strong>Conferences and Announcements</strong></a>   <a href="#other"><strong>Other</strong></a></p>

<p><br /></p>

<h2 id="book">Book</h2>

<ol class="bibliography"><li><span id="OSTHbook">Bezjak, S., Clyburne-Sherin, A., Conzett, P., Fernandes, P. L., Görögh, E., Helbig, K., Kramer, B., Labastida, I., Niemeyer, K., Psomopoulos, F. E., Ross-Hellauer, T., Schneider, R., Tennant, J., &amp; Verbakel, E. (2018). <i>The Open Science Training Handbook</i>. https://doi.org/10.5281/zenodo.1212495</span></li></ol>
<p><br /></p>

<h2 id="book-chapter">Book Chapter</h2>

<ol class="bibliography"><li><span id="DataMiningProteomicsGrid">Psomopoulos, F. E., &amp; Mitkas, P. A. (2009). Data Mining in Proteomics using Grid Computing. In M. Cannataro (Ed.), <i>Handbook of Research on Computational Grid Technologies for LifeSciences, Biomedicine and Healthcare</i> (pp. 245–267). IGI Global. https://doi.org/10.4018/978-1-60566-374-6</span></li></ol>

<p><br /></p>

<h2 id="journals">Journals</h2>

<p style="margin-bottom:15px"></p>

<h3 id="2020">2020</h3>

<ol class="bibliography"><li><span id="RedoxSPetrak">Stamatia, L., Alanis-Lobato, G., Pribyl, J., Raskó, T., Tichy, B., Mikulasek, K., Tsagiopoulou, M., Oppelt, J., Kastrinaki, G., Lefaki, M., Singh, M., Zink, A., Chondrogianni, N., Psomopoulos, F., Prigione, A., Ivics, Z., Pospisilova, S., Skladal, P., Izsvák, Z., … Petrakis, S. (2020). Nuclear inclusions of pathogenic ataxin-1 induce oxidative stress and perturb the protein synthesis machinery. <i>Redox Biology</i>, <i>32</i>, 101458. https://doi.org/https://doi.org/10.1016/j.redox.2020.101458</span></li>
<li><span id="Tsagiopoulouhaematol.2019.240721">Tsagiopoulou, M., Chapaprieta, V., Duran-Ferrer Martı́, Moysiadis, T., Psomopoulos, F., Kollia, P., Papakonstantinou, N., Campo, E., Stamatopoulos, K., &amp; Martin-Subero, J. I. (2020). Chronic lymphocytic leukemias with trisomy 12 show a distinct DNA methylation profile linked to altered chromatin activation. <i>Haematologica</i>. https://doi.org/10.3324/haematol.2019.240721</span></li>
<li><span id="Agathangelidishaematol.2019.221275">Agathangelidis, A., Galigalidou, C., Scarfò, L., Moysiadis, T., Rovida, A., Vlachonikola, E., Sofou, E., Psomopoulos, F., Vardi, A., Ranghetti, P., Siorenta, A., Galanis, A., Stamatopoulos, K., Chatzidimitriou, A., &amp; Ghia, P. (2020). High-throughput analysis of the T cell receptor gene repertoire in low-count monoclonal B cell lymphocytosis reveals a distinct profile from chronic lymphocytic leukemia. <i>Haematologica</i>. https://doi.org/10.3324/haematol.2019.221275</span></li>
<li><span id="Gavriilaki_2020">Gavriilaki, E., Sakellari, T. T. I., Batsis, I., Psomopoulos, F., Tsagiopoulou, M., Koutra, M., Yannaki, E., Papalexandri, A., Taylor, P., Nikolousis, E., Stamouli, M., Holbro, A., Baltadakis, I., Liga, M., Spyridonidis, A., Tsirigotis, P., Charchalakis, N., Tsakiris, D., Brodsky, R., … Anagnostopoulos, A. (2020). Pretransplant Genetic Susceptibility: Clinical Relevance in Transplant-Associated Thrombotic Microangiopathy. <i>Thrombosis and Haemostasis</i>, <i>120</i>(04), 638–646. https://doi.org/10.1055/s-0040-1702225</span></li>
<li><span id="KotouzaJCC_2020">Kotouza, M. T., Psomopoulos, F. E., &amp; Mitkas, P. A. (2020). A dockerized framework for hierarchical frequency-based document clustering on cloud computing infrastructures. <i>Journal of Cloud Computing</i>, <i>9</i>(2), 1–17. https://doi.org/10.1186/s13677-019-0150-y</span></li></ol>

<h3 id="2019">2019</h3>

<ol class="bibliography"><li><span id="FAIR_Software">Lamprecht, A.-L., Garcia, L., Kuzak, M., Martinez, C., Arcila, R., Martin Del Pico, E., Dominguez Del Angel, V., van de Sandt, S., Ison, J., Martinez, P. A., McQuilton, P., Valencia, A., Harrow, J., Psomopoulos, F., Gelpi, J. L., Chue Hong, N., Goble, C., &amp; Capella-Gutierrez, S. (2019). Towards FAIR principles for research software. <i>Data Science</i>, <i>2</i>(2), 1–23. https://doi.org/10.3233/DS-190026</span></li>
<li><span id="BISS_2019_ELIXIR">Kuzak, M., Harrow, J., Martinez, P. A., Psomopoulos, F. E., &amp; Via, A. (2019). ELIXIR Europe on the Road to Sustainable Research Software. <i>Biodiversity Information Science and Standards</i>, <i>3</i>, e37677. https://doi.org/10.3897/biss.3.37677</span></li>
<li><span id="PARLAPANI2019">Parlapani, F. F., Michailidou, S., Anagnostopoulos, D. A., Koromilas, S., Kios, K., Pasentsis, K., Psomopoulos, F., Argiriou, A., Haroutounian, S. A., &amp; Boziaris, I. S. (2019). Bacterial communities and potential spoilage markers of whole blue crab (Callinectes sapidus) stored under commercial simulated conditions. <i>Food Microbiology</i>, <i>82</i>, 325–333. https://doi.org/10.1016/j.fm.2019.03.011</span></li>
<li><span id="Kintsakis2019">Kintsakis, A. M., Psomopoulos, F. E., &amp; Mitkas, P. A. (2019). Reinforcement Learning based scheduling in a workflow management system. <i>Engineering Applications of Artificial Intelligence</i>, <i>81</i>, 94–106. https://doi.org/10.1016/j.engappai.2019.02.013</span></li>
<li><span id="Agathangelidis2019">Agathangelidis, A., Psomopoulos, F., &amp; Stamatopoulos, K. (2019). Stereotyped B Cell Receptor Immunoglobulins in B Cell Lymphomas. <i>Methods in Molecular Biology: "Lymphoma: Methods and Protocols"</i>, 139–155. https://doi.org/10.1007/978-1-4939-9151-8_7</span></li>
<li><span id="DataJournalDDPIG">Wu, M., Psomopoulos, F., Khalsa, S. J., &amp; de Waard, A. (2019). Data Discovery Paradigms: User Requirements and Recommendations for Data Repositories. <i>Data Science Journal</i>, <i>18</i>(1), 13. https://doi.org/10.5334/dsj-2019-003</span></li></ol>

<h3 id="2018">2018</h3>

<ol class="bibliography"><li><span id="gavriilaki2018pre">Gavriilaki, E., Touloumenidou, T., Sakellari, I., Batsis, I., Mallouri, D., Psomopoulos, F., Kotouza, M., Koutra, M., Yannaki, E., Papalexandri, A., Stamouli, M., Holbro, A., Baltadakis, I., Liga, M., Spyridonidis, A., Tsirigotis, P., Charchalakis, N., Passweg, J. R., Stamatopoulos, K., &amp; Anagnostopoulos, A. (2018). Pre-Transplant Genetic Susceptibility in Adult Allogeneic Hematopoietic Cell Transplant Recipients: Incidence and Clinical Relevance in Transplant-Associated Thrombotic Microangiopathy. <i>Blood</i>, <i>132</i>(Suppl 1), 3401–3401. https://doi.org/10.1182/blood-2018-99-112041</span></li>
<li><span id="vardi2018longitudinal">Vardi, A., Vlachonikola, E., Ioannou, N., Psomopoulos, F., Kotta, K., Papazoglou, D., Kotouza, M., Koravou, E., Galigalidou, C., Gemenetzi, K., Pasentsis, K., Iskas, M., Stavroyianni, N., Anagnostopoulos, A., Ramsay, A. G., Stamatopoulos, K., &amp; Hadzidimitriou, A. (2018). Longitudinal High-Throughput T Cell Repertoire Profiling of Chronic Lymphocytic Leukemia Patients Under Different Types of Treatment: Implications for Combination Strategies. <i>Blood</i>, <i>132</i>(Suppl 1), 4400–4400. https://doi.org/10.1182/blood-2018-99-118006</span></li>
<li><span id="IJC2018TP63">Papakonstantinou, N., Ntoufa, S., Tsagiopoulou, M., Moysiadis, T., Bhoi, S., Malousi, A., Psomopoulos, F., Mansouri, L., Laidou, S., Papazoglou, D., Gounari, M., Pasentsis, K., Plevova, K., Kuci-Emruli, V., Duran-Ferrer, M., Davis, Z., Ek, S., Rossi, D., Gaidano, G., … Stamatopoulos, K. (2018). Integrated epigenomic and transcriptomic analysis reveals TP63 as a novel player in clinically aggressive chronic lymphocytic leukemia. <i>International Journal of Cancer</i>, <i>0</i>(ja). https://doi.org/10.1002/ijc.31999</span></li>
<li><span id="FoodMicrobiology2018">Parlapani, F. F., Michailidou, S., Anagnostopoulos, D. A., Sakellariou, A. K., Pasentsis, K., Psomopoulos, F., Argiriou, A., Haroutounian, S. A., &amp; Boziaris, I. S. (2018). Microbial spoilage investigation of thawed common cuttlefish (Sepia officinalis) stored at 2 °C using next generation sequencing and volatilome analysis. <i>Food Microbiology</i>, <i>76</i>, 518–525. https://doi.org/10.1016/j.fm.2018.08.004</span></li>
<li><span id="ElevenTips">Gregory, K., Khalsa, S. J., Michener, W. K., Psomopoulos, F. E., de Waard, A., &amp; Wu, M. (2018). Eleven quick tips for finding research data. <i>PLOS Computational Biology</i>, <i>14</i>(4), 1–7. https://doi.org/10.1371/journal.pcbi.1006038</span></li></ol>

<h3 id="2017">2017</h3>

<ol class="bibliography"><li><span id="Hermes2017">Kintsakis, A. M., Psomopoulos, F. E., Symeonidis, A. L., &amp; Mitkas, P. A. (2017). Hermes: Seamless delivery of containerized bioinformatics workflows in hybrid cloud (HTC) environments. <i>SoftwareX</i>, <i>6</i>(Supplement C), 217–224. https://doi.org/10.1016/j.softx.2017.07.007</span></li>
<li><span id="SoilMetagenomics2017">Kalivas, A., Ganopoulos, I., Psomopoulos, F., Grigoriadis, I., Xanthopoulou, A., Hatzigiannakis, E., Osathanunkul, M., Tsaftaris, A., &amp; Madesis, P. (2017). Comparative metagenomics reveals alterations in the soil bacterial community driven by N-fertilizer and Amino 16® application in lettuce. <i>Genomics Data</i>, <i>14</i>(Supplement C), 14–17. https://doi.org/10.1016/j.gdata.2017.07.013</span></li>
<li><span id="XANTHOPOULOU201750">Xanthopoulou, A., Ganopoulos, I., Psomopoulos, F., Manioudaki, M., Moysiadis, T., Kapazoglou, A., Osathanunkul, M., Michailidou, S., Kalivas, A., Tsaftaris, A., Nianiou-Obeidat, I., &amp; Madesis, P. (2017). De novo comparative transcriptome analysis of genes involved in fruit morphology of pumpkin cultivars with extreme size difference and development of EST-SSR markers. <i>Gene</i>, <i>622</i>(Supplement C), 50–66. https://doi.org/10.1016/j.gene.2017.04.035</span></li>
<li><span id="RAPPRobotics2017">Zieliński, C., Stefańczyk, M., Kornuta, T., Figat, M., Dudek, W., Szynkiewicz, W., Kasprzak, W., Figat, J., Szlenk, M., Winiarski, T., Banachowicz, K., Zielińska, T., Tsardoulias, E. G., Symeonidis, A. L., Psomopoulos, F. E., Kintsakis, A. M., Mitkas, P. A., Thallas, A., Reppou, S. E., … Iturburu, M. (2017). Variable structure robot control systems: The RAPP approach. <i>Robotics and Autonomous Systems</i>, <i>94</i>(Supplement C), 226–244. https://doi.org/10.1016/j.robot.2017.05.002</span></li>
<li><span id="BioPaxViz2017">Psomopoulos, F. E., Vitsios, D. M., Baichoo, S., &amp; Ouzounis, C. A. (2017). BioPAXViz: a cytoscape application for the visual exploration of metabolic pathway evolution. <i>Bioinformatics</i>, <i>33</i>(9), 1418–1420. https://doi.org/10.1093/bioinformatics/btw813</span></li>
<li><span id="Tsardoulias2017">Tsardoulias, E. G., Kintsakis, A. M., Panayiotou, K., Thallas, A. G., Reppou, S. E., Karagiannis, G. T., Iturburu, M., Arampatzis, S., Zielinski, C., Prunet, V., Psomopoulos, F. E., Symeonidis, A. L., &amp; Mitkas, P. A. (2017). Towards an integrated robotics architecture for social inclusion – The RAPP paradigm. <i>Journal of Cognitive Systems Research</i>, <i>43</i>, 157–173. https://doi.org/10.1016/j.cogsys.2016.08.004</span></li></ol>

<h3 id="2016">2016</h3>

<ol class="bibliography"><li><span id="KintsakisBigData2016">Kintsakis, A. M., Psomopoulos, F. E., &amp; Mitkas, P. A. (2016). Data-aware optimization of bioinformatics workflows in hybrid clouds. <i>Journal of Big Data</i>, <i>3</i>(1), 20. https://doi.org/10.1186/s40537-016-0055-2</span></li>
<li><span id="PlantOmics2016">Zambounis, A., Psomopoulos, F. E., Ganopoulos, I., Avramidou, E., Aravanopoulos, F. A., Tsaftaris, A., &amp; Madesis, P. (2016). In silico analysis of the LRR receptor-like serine threonine kinases subfamily in Morus notabilis. <i>Plant Omics Journal</i>, <i>9</i>(5), 319–326. https://doi.org/10.21475/poj.09.05.16.pne126</span></li>
<li><span id="Reppou2016">Reppou, S. E., Tsardoulias, E. G., Kintsakis, A. M., Symeonidis, A. L., Mitkas, P. A., Psomopoulos, F. E., Karagiannis, G. T., Zielinski, C., Prunet, V., Merlet, J.-P., Iturburu, M., &amp; Gkiokas, A. (2016). RAPP: A Robotic-Oriented Ecosystem for Delivering Smart User Empowering Applications for Older People. <i>International Journal of Social Robotics</i>, 1–14. https://doi.org/10.1007/s12369-016-0361-z</span></li>
<li><span id="Chatzidimopoulos2016">Chatzidimopoulos, M., Psomopoulos, F., Malandrakis, E. E., Ganopoulos, I., Madesis, P., Vellios, E. K., &amp; Drogoudi, P. (2016). Comparative Genomics of Botrytis cinerea Strains with Differential Multi-Drug Resistance. <i>Frontiers in Plant Science</i>, <i>7</i>(April), 1–3. https://doi.org/10.3389/fpls.2016.00554</span></li>
<li><span id="Xanthopoulou2016">Xanthopoulou, A., Psomopoulos, F., Ganopoulos, I., Manioudaki, M., Tsaftaris, A., Nianiou-Obeidat, I., &amp; Madesis, P. (2016). De novo transcriptome assembly of two contrasting pumpkin cultivars. <i>Genomics Data</i>, <i>7</i>, 200–201. https://doi.org/10.1016/j.gdata.2016.01.006</span></li></ol>

<h3 id="2015">2015</h3>

<ol class="bibliography"><li><span id="Duarte2015">Duarte, A. M. S., Psomopoulos, F. E., Blanchet, C., Bonvin, A. M. J. J., Corpas, M., Franc, A., Jimenez, R. C., de Lucas, J. M., Nyrönen, T., Sipos, G., &amp; Suhr, S. B. (2015). Future opportunities and trends for e-infrastructures and life sciences: going beyond the grid to enable life science data analysis. <i>Frontiers in Genetics</i>, <i>6</i>(June), 197. https://doi.org/10.3389/fgene.2015.00197</span></li>
<li><span id="Vitsios2015">Vitsios, D. M., Psomopoulos, F. E., Mitkas, P. a., &amp; Ouzounis, C. a. (2015). Inference of Pathway Decomposition Across Multiple Species Through Gene Clustering. <i>International Journal on Artificial Intelligence Tools</i>, <i>24</i>(01), 1540003. https://doi.org/10.1142/S0218213015400035</span></li></ol>

<h3 id="2013">2013</h3>

<ol class="bibliography"><li><span id="FuzzyBMCBioinformatics">Psomopoulos, F. E., Mitkas, P. A., &amp; Ouzounis, C. A. (2013). Detection of genomic idiosyncrasies using fuzzy phylogenetic profiles. <i>PloS One</i>, <i>8</i>(1), e52854. https://doi.org/10.1371/journal.pone.0052854</span></li></ol>

<h3 id="2012">2012</h3>

<ol class="bibliography"><li><span id="GenesChlamydiae">Psomopoulos, F. E., Siarkou, V. I., Papanikolaou, N., Iliopoulos, I., Tsaftaris, A. S., Promponas, V. J., &amp; Ouzounis, C. A. (2012). The chlamydiales pangenome revisited: structural stability and functional coherence. <i>Genes</i>, <i>3</i>(2), 291–319. https://doi.org/10.3390/genes3020291</span></li>
<li><span id="MultiLevelClusteringJournal">Psomopoulos, F. E., &amp; Mitkas, P. A. (2012). Multi-Level Clustering of Phylogenetic Profiles. <i>International Journal on Artificial Intelligence Tools (Special Issue on Bioinformatics and Bioengineering)</i>, <i>21</i>(5), 1240023. https://doi.org/10.1142/S0218213012400234</span></li></ol>

<h3 id="2010">2010</h3>

<ol class="bibliography"><li><span id="Psomopoulos2010a">Psomopoulos, F. E., &amp; Mitkas, P. A. (2010). Bioinformatics algorithm development for Grid environments. <i>Journal of Systems and Software</i>, <i>83</i>(7), 1249–1257. https://doi.org/10.1016/j.jss.2010.01.051</span></li></ol>

<h3 id="2009">2009</h3>

<ol class="bibliography"><li><span id="gMolKnotMolSimul">Psomopoulos, F. E., Mitkas, P. A., Krinas, C. S., &amp; Demetropoulos, I. N. (2009). A grid-enabled algorithm yields figure-eight molecular knot. <i>Molecular Simulation</i>, <i>35</i>(9), 725–736. https://doi.org/10.1080/08927020902833103</span></li></ol>

<p><br /></p>

<h2 id="conferences-and-announcements">Conferences and Announcements</h2>

<p style="margin-bottom:15px"></p>

<!-- ### 2020

<ol class="bibliography"></ol> -->

<h3 id="2019-1">2019</h3>

<ol class="bibliography"><li><span id="ADBIS2019">Kotouza, M. T., Psomopoulos, F. E., &amp; Mitkas, P. A. (2019). A Dockerized String Analysis Workflow for Big Data. <i>23rd European Conference on Advances in Databases and Information Systems, ASBIS 2019, Bled, Slovenia, September 8-11, 2019</i>, 564–569. https://doi.org/10.1007/978-3-030-30278-8_55</span></li>
<li><span id="FAIR_Softwarf">Vardi, A., Vlachonikola, E., Mourati, S., Psomopoulos, F., Pantouloufos, N., Kouvatsi, A., Stavroyianni, N., Anagnostopoulos, A., Stamatopoulos, K., &amp; Hadzidimitriou, A. (2019). PS1131 High-Throughput B-Cell immunoprofiling at diagnosis and relapse offers further evidence of functional selection throughout the natural history of chronic lymphocytic leukemia. <i>HemaSphere</i>, <i>3</i>(S1), 512. https://doi.org/10.1097/01.HS9.0000562808.48237.52</span></li>
<li><span id="BloodASH_2019d">Gemenetzi, K., Agathangelidis, A., Psomopoulos, F., Pasentsis, K., Koravou, E., Iskas, M., Stavroyianni, N., Anagnostopoulos, A., Sandaltzopoulos, R., Stamatopoulos, K., &amp; Chatzidimitriou, A. (2019). VH CDR3-Focused Somatic Hypermutation in CLL IGHV-IGHD-IGHJ Gene Rearrangements with 100\% IGHV Germline Identity. <i>Blood</i>, <i>134</i>(Supplement_1), 4277–4277. https://doi.org/10.1182/blood-2019-127979</span></li>
<li><span id="BloodASH_2019c">Gerousi, M., Psomopoulos, F., Kotta, K., Stavroyianni, N., Anagnostopoulos, A., Kotsianidis, I., Ntoufa, S., &amp; Stamatopoulos, K. (2019). Functional Calcitriol/Vitamin D Receptor Signaling in Chronic Lymphocytic Leukemia. <i>Blood</i>, <i>134</i>(Supplement_1), 3019–3019. https://doi.org/10.1182/blood-2019-127910</span></li>
<li><span id="BloodASH_2019b">Gemenetzi, K., Agathangelidis, A., Psomopoulos, F., Plevova, K., Sutton, L.-A., Pasentsis, K., Anagnostopoulos, A., Sandaltzopoulos, R., Rosenquist, R., Davi, F., Pospisilova, S., Stamatopoulos, K., &amp; Chatzidimitriou, A. (2019). Higher Order Restrictions of the Immunoglobulin Repertoire in CLL: The Illustrative Case of Stereotyped Subsets #2 and #169. <i>Blood</i>, <i>134</i>(Supplement_1), 5453–5453. https://doi.org/10.1182/blood-2019-128017</span></li>
<li><span id="BloodASH_2019a">Tsagiopoulou, M., Chapaprieta, V., Russiñol, N., Psomopoulos, F., Papakonstantinou, N., Stavroyianni, N., Anagnostopoulos, A., Kollia, P., Campo, E., Stamatopoulos, K., &amp; Martin-Subero, J. I. (2019). Genome-Wide Histone Acetylation Profiling in Chronic Lymphocytic Leukemia Reveals a Distinctive Signature in Stereotyped Subset #8. <i>Blood</i>, <i>134</i>(Supplement_1), 1241–1241. https://doi.org/10.1182/blood-2019-127817</span></li></ol>

<h3 id="2018-1">2018</h3>

<ol class="bibliography"><li><span id="BDCAT2018">Kotouza, M. T., Vavliakis, K. N., Psomopoulos, F. E., &amp; Mitkas, P. A. (2018). A hierarchical multi-metric framework for item clustering. <i>5th IEEE/ACM International Conference on Big Data Computing Applications and Technologies, BDCAT 2018, Zurich, Switzerland, December 17-20, 2018</i>, 191–197. https://doi.org/10.1109/BDCAT.2018.00031</span></li>
<li><span id="WSSSPE61">Kuzak, M., Harrow, J., Jimenez, R. C., Martinez, P., Psomopoulos, F. E., Svobodová Vařeková, R., &amp; Via, A. (2018). Lesson development for Open Source Software best practices adoption. <i>Working Towards Sustainable Software for Science Practice and Experiences</i>. https://doi.org/10.5281/zenodo.1420462</span></li>
<li><span id="14thAIAI">Tsarouchis, S.-F., Kotouza, M. T., Psomopoulos, F. E., &amp; Mitkas, P. A. (2018). A Multi-metric Algorithm for Hierarchical Clustering of Same-Length Protein Sequences. <i>7th Mining Humanistic Data Workshop (MHDW2018) in Conjunction with the 14th International Conference on Artificial Intelligence Applications and Innovations (AIAI 2018)</i>, 189–199. https://doi.org/10.1007/978-3-319-92016-0_18</span></li></ol>

<h3 id="2017-1">2017</h3>

<ol class="bibliography"><li><span id="ECCB2017Pangenomes">Mpatziakas, A., Psomopoulos, F. E., Moysiadis, T., &amp; Sgardelis, S. (2017). Computing pangenome statistics in R, [v1; not peer reviewed]. <i>F1000research 2017 (5th Annual International Conference on Intelligent Systems for Molecular Biology (ISMB,) / 16th European Conference on Computational Biology (ECCB))</i>, <i>6(ISCB Comm J)</i>(1529), 1. https://doi.org/10.7490/f1000research.1114765.1</span></li>
<li><span id="DI4R2017">Culina, A., Kuzak, M., &amp; Psomopoulos, F. E. (2017). Training Life Scientists in Open Reproducible Research; a Data Science scenario. <i>Digital Infrastructures for Research 2017</i>, 1–2. https://doi.org/10.6084/m9.figshare.5644279.v1</span></li>
<li><span id="gemenetzi2017high">Gemenetzi, K., Stalika, E., Vardi, A., Psomopoulos, F. E., Minga, E., Anagnostopoulos, A., Stamatopoulos, K., Hadzidimitriou, A., &amp; Sutton, L. A. (2017). HIGH THROUGHPUT IMMUNOPROFILING OF CHRONIC LYMPHOCYTIC LEUKEMIA PATIENTS ASSIGNED TO STEREOTYPED SUBSET# 4: NOVEL INSIGHTS INTO THE DEPTH, DIVERSITY AND TEMPORAL DYNAMICS OF CLONAL EVOLUTION. <i>HAEMATOLOGICA</i>, <i>102</i>, 67–67.</span></li>
<li><span id="BloodASH_2017b">Agathangelidis, A., Galigalidou, C., Scarfo, L., Rovida, A., Gounari, M., Psomopoulos, F., Ranghetti, P., Gemenetzi, K., Davi, F., Stamatopoulos, K., Hadzidimitriou, A., &amp; Ghia, P. (2017). CLL-Specific B Cell Receptor Immunoglobulin Stereotypes Are Very Infrequent in Low Count Monoclonal B Cell Lymphocytosis: Implications for Disease Ontogeny. <i>Blood</i>, <i>130</i>(Suppl 1), 1710–1710. http://www.bloodjournal.org/content/130/Suppl_1/1710</span></li>
<li><span id="BloodASH_2017a">Agathangelidis, A., Galigalidou, C., Scarfo, L., Rovida, A., Moysiadis, T., Vlachonikola, E., Psomopoulos, F., Ranghetti, P., Vardi, A., Stamatopoulos, K., Hadzidimitriou, A., &amp; Ghia, P. (2017). High Throughput Immunoprofiling of Low Count B Lymphocytosis Reveals a Distinct T Cell Receptor Repertoire from Chronic Lymphocytic Leukemia. <i>Blood</i>, <i>130</i>(Suppl 1), 4296–4296. http://www.bloodjournal.org/content/130/Suppl_1/4296</span></li></ol>

<h3 id="2016-1">2016</h3>

<ol class="bibliography"><li><span id="HSBMB2016">Mokos, P., Psomopoulos, F. E., Ainali, C., Argiriou, A., Charalampidou, A., Hadzopoulou-Cladaras, M., &amp; Dafou, D. (2016). Network analysis and regularized regression modeling identifies key regulatory molecular events underlying hepatocellular carcinoma dedifferentiation. <i> 67th Annual Conference of the Hellenic Society for Biochemistry and Molecular Biology (HSBMB)</i>, 1–4.</span></li>
<li><span id="HeCrA16">Mokos, P., Psomopoulos, F. E., Ainali, C., Argiriou, A., Charalampidou, A., Hadzopoulou-Cladaras, M., &amp; Dafou, D. (2016). Integrated network analysis and regularized regression modeling reveal regulatory miRNA-mRNA networks associated with hepatocellular carcinoma dedifferentiation. <i>8th Conference of the Hellenic Crystallographic Association (HeCrA) / 11th Conference of the Hellenic Society for Computational Biology and Bioinformatics (HeCrA-HSCBB16)</i>, 1–2.</span></li>
<li><span id="PlantBreedingXanthopoulou">Xanthopoulou, A., Psomopoulos, F., Ganopoulos, I., Kalyvas, A., Tsaftaris, A., Madesis, P., &amp; Nianiou-Obeidat, I. (2016). De novo sequencing and comparative analysis of the transcriptome of two Cucurbita pepo cultivars (in Greek). <i>16th Conference of the Hellenic Plant Breeding Society</i>, 1–2.</span></li>
<li><span id="DI4R2016">Via, A., Fernandes, P., Korpelainen, E., Duarte, A. M. S., &amp; Psomopoulos, F. E. (2016). E-Infrastructures for the Bioinformatics long tail of science: a user perspective. <i>Digital Infrastructures for Research 2016</i>, 1–2.</span></li>
<li><span id="8thRDA">Psomopoulos, F. E. (2016). Managing the Data Ecosystem in Life Sciences through Bioinformatics. <i>8th RDA Plenary</i>, 1–1.</span></li>
<li><span id="12thAIAI">Stergiadis, E., Kintsakis, A. M., Psomopoulos, F. E., &amp; Mitkas, P. A. (2016). A scalable Grid Computing framework for extensible phylogenetic profile construction. <i>5th Mining Humanistic Data Workshop (MHDW2016) in Conjunction with the 12th International Conference on Artificial Intelligence Applications and Innovations (AIAI 2016)</i>, 1–8.</span></li>
<li><span id="ECCB2016Pangenomes">Psomopoulos, F. E., Kintsakis, A. M., &amp; Mitkas, P. A. (2016). A pan-genome approach and application to species with photosynthetic capabilities, [v1; not peer reviewed]. <i>F1000Research 2016</i>, <i>5(ECCB 2016</i>(2132), 1. https://doi.org/10.7490/f1000research.1112964.1</span></li>
<li><span id="ECCB2016FedCloud">Psomopoulos, F. E., Korpelainen, E., Mattila, K., &amp; Scardaci, D. (2016). Bioinformatics resources on EGI Federated Cloud, [v1; not peer reviewed]. <i>F1000Research 2016</i>, <i>5(ECCB 2016</i>(2131), 1. https://doi.org/10.7490/f1000research.1112963.1</span></li></ol>

<h3 id="2015-1">2015</h3>

<ol class="bibliography"><li><span id="26thPHCErrorCorrection">Kintsakis, A., Vrousgou, O., Vardi, A., Karypidou, M., Stalika, E., Korabou, E., P Zerva, P., Anagnostopoulos, A., Chatzidimitriou, A., Stamatopoulos, K., &amp; Psomopoulos, F. E. (2015). Large-scale detection of sequencing errors using immunogenetic data of β chain T cell receptors (in Greek). <i>26th Panhellenic Haematology Congress</i>, 1–2.</span></li>
<li><span id="26thPHCAward">Ntoufa, S., Papakonstantinou, N., Tsagiopoulou, M., Moisiadis, T., Malousi, A., Papazoglou, D., Pasentsis, K., Psomopoulos, F., Stalika, E., Laidou, M., Maglaveras, N., Anagnostopoulos, A., &amp; Stamatopoulos, K. (2015). Distinct methylation patterns in subgroups of patients with Chronic Lymphocytic Leukemia reveal TP63 as a new player in clinical aggressiveness ("Arkagathos Gouttas" Prize award). <i>26th Panhellenic Haematology Congress</i>, 1–2.</span></li>
<li><span id="EEBMB2015">Mokos, P., Psomopoulos, F. E., Ainali, C., Charalampidou, A., Hadzopoulou-Cladaras, M., &amp; Dafou, D. (2015). Large integrative analysis for the identification of novel molecular targets in hepatocellular carcinoma. <i>66th Congress of the Hellenic Society of Biochemistry and Molecular Biology</i>, 1–4.</span></li>
<li><span id="Duarte2015a">Duarte, A., Psomopoulos, F. E., Sipos, G., Ferreira, N., &amp; Scardaci, D. (2015). EGI Support for Genome Analysis and Protein Folding - A Virtual Team for Life Sciences. <i>International Symposium on Grids and Clouds ( ISGC ) 2015</i>, 2.</span></li>
<li><span id="Psomopoulos2015">Psomopoulos, F. E., Vrousgou, O. T., &amp; Mitkas, P. A. (2015). Large-scale modular comparative genomics: the Grid approach [v1; not peer reviewed]. <i>F1000Research 2015</i>, <i>4(ISCB Com</i>(377), 1. https://doi.org/10.7490/f1000research.1110127.1</span></li>
<li><span id="Vrousgou2015">Vrousgou, O. T., Psomopoulos, F. E., &amp; Mitkas, P. A. (2015). A grid-enabled modular framework for efficient sequence analysis workflows. <i>16th International Conference on Engineering Applications of Neural Networks</i>, 10.</span></li></ol>

<h3 id="2014">2014</h3>

<ol class="bibliography"><li><span id="Psomopoulos2014">Psomopoulos, F. E., &amp; Mitkas, P. A. (2014). Algebraic Interpretations Towards Clustering Protein Homology Data. <i>Artificial Intelligence Applications and Innovations, IFIP Advances in Information and Communication Technology Volume 437</i>. https://doi.org/10.1007/978-3-662-44722-2_15</span></li>
<li><span id="Psomopoulos2014a">Psomopoulos, F. E., &amp; Ouzounis, C. A. (2014). Computation and visualization of ancestral pathway reconstruction and inference. <i>ECCB 2014</i>, <i>8</i>, 52854.</span></li>
<li><span id="Psomopoulos2014b">Psomopoulos, F., Tsardoulias, E., Giokas, A., Zielinski, C., Prunet, V., Trochidis, I., Daney, D., Serrano, M., Courtes, L., Arampatzis, S., &amp; Mitkas, P. A. (2014). RAPP System Architecture. <i>Workshop on Assistance and Service Robotics in a Human Environment (ASROB), in Conjunction with the IEEE/RSJ International Conference on Intelligent Robots and Systems (IROS’14)</i>, 8.</span></li></ol>

<h3 id="2012-1">2012</h3>

<ol class="bibliography"><li><span id="HSCBB12Gluco">Psomopoulos, F. E., Argiriou, A., Tsaftaris, A. S., &amp; Ouzounis, C. A. (2012). A Pangenome Analysis of the Glucosinolate Pathway in Plants. <i>7th Conference of the Hellenic Society for Computational Biology &amp; Bioinformatics - HSCBB ’12</i>, 63.</span></li>
<li><span id="HSCBB12Ancestral">Psomopoulos, F. E., &amp; Ouzounis, C. A. (2012). Ancestral Reconstruction of Metabolic Pathway Content at the Paleome Level. <i>7th Conference of the Hellenic Society for Computational Biology &amp; Bioinformatics - HSCBB ’12</i>, 90.</span></li>
<li><span id="SFHMMY5">Vitsios, D., Psomopoulos, F. E., Mitkas, P. A., &amp; Ouzounis, C. A. (2012). Detection of closely correlated genes across genomes using pathway data. <i>5th National Student Conference of Electrical and Computer Engineering</i>, 111–117.</span></li>
<li><span id="HSCBB12BioPax">Vitsios, D., Psomopoulos, F. E., &amp; Ouzounis, C. A. (2012). Null Entries for BioPAX Pathways, Visualization and Pathway Phylogenetic Profiling Based on Paxtools. <i>7th Conference of the Hellenic Society for Computational Biology &amp; Bioinformatics - HSCBB ’12</i>, 92.</span></li></ol>

<h3 id="2011">2011</h3>

<ol class="bibliography"><li><span id="HSCBB11">Vitsios, D., Psomopoulos, F. E., Mitkas, P. A., &amp; Ouzounis, C. A. (2011). Detecting species evolution through metabolic pathways. <i>6th Conference of the Hellenic Society For Computational Biology and Bioinformatics - HSCBB ’11</i>, 16.</span></li></ol>

<h3 id="2010-1">2010</h3>

<ol class="bibliography"><li><span id="KyrchaPsomopoulosEGEEForum">Chatzidimitriou, K. C., Psomopoulos, F. E., &amp; Mitkas, P. A. (2010). Grid-enabled parameter initialization for high performance machine learning tasks. <i>5th EGEE User Forum</i>, 113–114.</span></li>
<li><span id="Psomopoulos2010">Psomopoulos, F. E., &amp; Mitkas, P. A. (2010). Multi Level Clustering of Phylogenetic Profiles. <i>10th IEEE International Conference on Bioinformatics and Bioingineering</i>, 308–309. https://doi.org/10.1109/BIBE.2010.67</span></li>
<li><span id="HSCBB10">Psomopoulos, F. E., Mitkas, P. A., &amp; Ouzounis, C. A. (2010). Clustering of discrete and fuzzy phylogenetic profiles. <i>5th Conference of the Hellenic Society For Computational Biology and Bioinformatics - HSCBB ’10</i>, 58.</span></li>
<li><span id="TzimaPsomopoulosEGEEForum">Tzima, F. A., Psomopoulos, F. E., &amp; Mitkas, P. A. (2010). An investigation of the effect of clustering-based initialization on Learning Classifiers Systems’ effectiveness: leveraging the Grid infrastructure. <i>5th EGEE User Forum</i>, 111–112.</span></li></ol>

<h3 id="2009-1">2009</h3>

<ol class="bibliography"><li><span id="HSCBB09">Karagiannis, K. M., Psomopoulos, F. E., &amp; Mitkas, P. A. (2009). Multi Level Clustering of Phylogenetic Profiles. <i>4th Conference of the Hellenic Society For Computational Biology and Bioinformatics - HSCBB ’09</i>, 1.</span></li>
<li><span id="FpsomPCI2009">Psomopoulos, F. E., &amp; Mitkas, P. A. (2009). BADGE: Bioinformatics Algorithm Development for Grid Environments. <i>13th Panhellenic Conference on Informatics</i>, 93–107.</span></li>
<li><span id="ASSISTForum2009">Mitkas, P. A., Delopoulos, A., Vavliakis, K., Maramis, C., Falelakis, M., Psomopoulos, F. E., Koutkias, V., Lekka, I., Tantsis, A., Mikos, A., Maglaveras, N., &amp; Agorastos, T. (2009). Combining multiple sources towards cervical cancer prognosis – the ASSIST platform (in Greek). <i>8th Scientific Forum on "New Trends in Prognosis and Treatment of Cervical Cancer"</i>, 109–118.</span></li></ol>

<h3 id="2008">2008</h3>

<ol class="bibliography"><li><span id="WCCAssist">Agorastos, T., Mitkas, P. A., Falelakis, M., Psomopoulos, F. E., Delopoulos, A. N., Symeonidis, A. L., Diplaris, S., Maramis, C., Batzios, A., Lekka, I., Koutkias, V., Mikos, T., Tatsis, A., &amp; Maglaveras, N. (2008). Large Scale Association Studies Using Unified Data for Cervical Cancer and beyond: The ASSIST Project. <i>World Cancer Congress</i>, 1.</span></li>
<li><span id="GkekasPsomopoulosSFHMMY">Gkekas, C. N., Psomopoulos, F. E., &amp; Mitkas, P. A. (2008). A Parallel Data Mining Methodology for Protein Function Prediction Utilizing Finite State Automata. <i>2nd Electrical and Computer Engineering Student Conference</i>, 6.</span></li>
<li><span id="CkekasFpsomSamos">Gkekas, C. N., Psomopoulos, F. E., &amp; Mitkas, P. A. (2008). Exploiting parallel data mining processing for protein annotation. <i>Student EUREKA 2008: 2nd Panhellenic Scientific Student Conference</i>, 242–252.</span></li>
<li><span id="GkekasPsomopoulosEGEEForum">Gkekas, C. N., Psomopoulos, F. E., &amp; Mitkas, P. A. (2008). A Parallel Data Mining Application for Gene Ontology Term Prediction. <i>3rd EGEE User Forum</i>, 1.</span></li>
<li><span id="EsbmeAssist">Mitkas, P. A., Maramis, C., Delopoulos, A. N., Symeonidis, A. L., Diplaris, S., Falelakis, M., Psomopoulos, F. E., Batzios, A., Maglaveras, N., Lekka, I., Koutkias, V., Agorastos, T., Mikos, T., &amp; Tatsis, A. (2008). ASSIST: Employing Inference and Semantic Technologies to Facilitate Association Studies on Cervical Cancer. <i>6th European Symposium on Biomedical Engineering</i>.</span></li>
<li><span id="MprouzaFpsomSamos">Mprouza, I. K., Psomopoulos, F. E., &amp; Mitkas, P. A. (2008). AMoS: Agent-based Molecular Simulations. <i>Student EUREKA 2008: 2nd Panhellenic Scientific Student Conference</i>, 175–186.</span></li>
<li><span id="HSCBB08">Psomopoulos, F. E., &amp; Mitkas, P. A. (2008). Sizing Up: Bioinformatics in a Grid Context. <i>3rd Conference of the Hellenic Society For Computational Biology and Bioinformatics - HSCBB ’08</i>, 1.</span></li>
<li><span id="PsomopoulosDemetropoulosHellasGrid">Psomopoulos, F. E., Mitkas, P. A., Krinas, C. S., &amp; Demetropoulos, I. N. (2008). G-MolKnot: A grid enabled systematic algorithm to produce open molecular knots. <i>1st HellasGrid User Forum</i>, 24–25.</span></li></ol>

<h3 id="2007">2007</h3>

<ol class="bibliography"><li><span id="GkekasPsomopoulosBioacademy">Gkekas, C. N., Psomopoulos, F. E., &amp; Mitkas, P. A. (2007). Modeling Gene Ontology Terms using Finite State Automata. <i>Hellenic Bioinformatics and Medical Informatics Meeting</i>, 1.</span></li>
<li><span id="ASSISTBioacademy">Mitkas, P. A., Delopoulos, A. N., Symeonidis, A. L., &amp; Psomopoulos, F. E. (2007). A Framework for Semantic Data Integration and Inferencing on Cervical Cancer. <i>Hellenic Bioinformatics and Medical Informatics Meeting</i>, 1.</span></li>
<li><span id="MprouzaPsomopoulosBioacademy">Mprouza, I. K., Psomopoulos, F. E., &amp; Mitkas, P. A. (2007). Simulating molecular dynamics through intelligent software agents. <i>Hellenic Bioinformatics and Medical Informatics Meeting</i>, 1.</span></li></ol>

<h3 id="2006">2006</h3>

<ol class="bibliography"><li><span id="PolychroniadouPsomopoulosGClass">Polychroniadou, H. E., Psomopoulos, F. E., &amp; Mitkas, P. A. (2006). G-Class: A Divide and Conquer Application for Grid Protein Classification. <i>Proceedings of the 2nd ADMKD 2006: Workshop on Data Mining and Knowledge Discovery (in Conjunction with ADBIS 2006: The 10th East-European Conference on Advances in Databases and Information Systems)</i>, 121–132.</span></li>
<li><span id="PsomopoulosHDMS">Psomopoulos, F. E., &amp; Mitkas, P. A. (2006). PROTEAS: A Finite State Automata based data mining algorithm for rule extraction in protein classification. <i>Proceedings of the 5th Hellenic Data Management Symposium</i>, 118–126.</span></li></ol>

<h3 id="2005">2005</h3>

<ol class="bibliography"><li><span id="LoutrakiPsomopoulos">Psomopoulos, F. E., &amp; Mitkas, P. A. (2005). A protein classification engine based on stochastic finite state automata. <i>Lecture Series on Computer and Computational Sciences VSP/Brill (Proceedings of the Symposium 35: Computational Methods in Molecular Biology in Conjunction with ICCMSE)</i>, <i>4B</i>, 1371–1374.</span></li></ol>

<h3 id="2004">2004</h3>

<ol class="bibliography"><li><span id="PsomopoulosDiplarisPisa">Psomopoulos, F. E., Diplaris, S., &amp; Mitkas, P. A. (2004). A finite state automata based technique for protein classification rules induction. <i>Proceedings of the Second European Workshop on Data Mining and Text Mining in Bioinformatics (in Conjunction with ECML/PKDD)</i>, 54–60.</span></li></ol>

<p><br /></p>

<h2 id="other">Other</h2>

<p style="margin-bottom:15px"></p>

<h3 id="short-articles">Short articles</h3>

<ol class="bibliography"><li><span id="godel2019dna">Nicolaidis, A., &amp; Psomopoulos, F. (2019). <i>DNA coding and Gödel numbering</i>. https://arxiv.org/abs/1909.13574</span></li>
<li><span id="erin_alison_becker_2019_3260609">Becker, E. A., Reiter, T., Psomopoulos, F., McKay, S. J., Mizzi, J. E., Williams, J., Hoyt, P. R., Crosby, K., Nederbragt, L., Teal, T., Infante, A., Pierce, T., Michonneau, F., Augustus, G., Harris, R. M., dbmarchant, Thomas, A., Cranston, K., Weitemier, K., … Kretzschmar, W. (2019). <i>datacarpentry/wrangling-genomics: Data Carpentry:
                   Genomics data wrangling and processing, June 2019</i>. https://doi.org/10.5281/zenodo.3260609</span></li>
<li><span id="wilson_greg_2017_1064254">Wilson, G., Becker, E., McKay, S., Michonneau, F., Williams, J. J., Mayes, A. C., Silva, R., (hoytpr), P., Crosby, K., Nederbragt, L., NielInfante, Psomopoulos, F. E., Gourlé, H., (gaiusjaugustus), G., dbmarchant, Emonet, R., Cock, P., Banaszkiewicz, P., Teal, T., … gtehennepe. (2017). <i>Data Carpentry Wrangling Genomics Lesson</i>. https://doi.org/10.5281/zenodo.1064254</span></li>
<li><span id="Psomopoulos2016-RDAb">Psomopoulos, F. E. (2016). International Data Forum Opening Plenary Session: Maintaining Scientific Rigor and Enhancing Discovery. In <i>RDA Newsletter</i>. https://www.rd-alliance.org/blogs/international-data-forum-opening-plenary-session-maintaining-scientific-rigor-and-enhancing</span></li>
<li><span id="Psomopoulos2016-RDAa">Psomopoulos, F. E. (2016). International Data Forum: Data Stories Session. In <i>RDA Newsletter</i>. https://www.rd-alliance.org/blogs/international-data-forum-data-stories-session.html</span></li>
<li><span id="Psomopoulos2016-CSC">Psomopoulos, F. E. (2016). Training life scientists to analyse genomic data – can cloud make life easier? (interview). In <i>@CSC Online Magazine Tells More</i>. https://www.csc.fi/web/atcsc/-/training-life-scientists-to-analyse-genomic-data-can-cloud-make-life-easier</span></li>
<li><span id="Psomopoulos2015a">Psomopoulos, F., &amp; Jimenez, R. (2015). EGI-ELIXIR project: Integrating datasets for bioinformatics. In <i>EGI Inspire Newsletter #18</i> (Number February, pp. 5–6).</span></li>
<li><span id="Duarte2014">Duarte, A., &amp; Psomopoulos, F. (2014). Future opportunities and trends for e-infrastructures and life sciences. In <i>EGI Inspire Newsletter #14</i> (Number January, p. 1).</span></li>
<li><span id="Psomopoulos2008">Psomopoulos, F. E., Gkekas, C., &amp; Mitkas, P. A. (2008). Data Mining in Bioinformatics using Grid Computing (in Greek). In <i>AUTH Grid Team, Aristotle University of Thessaloniki</i> (Number January, p. 1).</span></li></ol>

<h3 id="thesis">Thesis</h3>

<ol class="bibliography"><li><span id="FpsomPhDThesis">Psomopoulos, F. E. (2010). <i>Parallel Data Mining and Analysis Algorithms in a Grid environment and applications in Bioinformatics</i> (p. 184) [PhD Thesis]. Aristotle University of Thessaloniki, Greece.</span></li>
<li><span id="FpsomMasterThesis">Psomopoulos, F. E. (2004). <i>A finite state automata algorithm for the extraction of association rules and applications in protein data classification (Greek text only)</i> (p. 108) [Master Thesis]. Aristotle University of Thessaloniki, Greece.</span></li></ol>



  </div>
</div>


      </div>
    </main>

    <footer>
  <div class="container">
    <div class="row">
      <div class="col-lg">
        <ul class="list-inline">
          
            <li class="list-inline-item">
              <a href="mailto:fpsom@certh.gr">
                <span class="fa-stack fa-lg">
                  <i class="fa fa-circle fa-stack-2x"></i>
                  <i class="fa fa-envelope-o fa-stack-1x fa-inverse"></i>
                </span>
              </a>
            </li>
          

          
            <li class="list-inline-item">
              <a href="https://twitter.com/fopsom">
                <span class="fa-stack fa-lg">
                  <i class="fa fa-circle fa-stack-2x"></i>
                  <i class="fa fa-twitter fa-stack-1x fa-inverse"></i>
                </span>
              </a>
            </li>
          

          
            <li class="list-inline-item">
              <a href="https://github.com/fpsom">
                <span class="fa-stack fa-lg">
                  <i class="fa fa-circle fa-stack-2x"></i>
                  <i class="fa fa-github fa-stack-1x fa-inverse"></i>
                </span>
              </a>
            </li>
          
        </ul>
      </div>

      
      <div class="col-lg">
        


        
          <ul class="fa-ul">
          
            <li>
              <i class="fa-li fa fa-chevron-circle-right"></i>
              <a href="https://inab.certh.gr/" target="_blank">Institute of Applied Biosciences (INAB)</a>
            </li>
          
            <li>
              <i class="fa-li fa fa-chevron-circle-right"></i>
              <a href="https://www.certh.gr/" target="_blank">Centre for Research and Technology Hellas (CERTH)</a>
            </li>
          
          </ul>
        

      </div>
      
      <div class="col-lg">
        <p>This is the website for our research group</p>


        

      </div>
      
    </div>

    <div class="small mt-2">
      
        <p>© Fotis Psomopoulos</p>
      
      
        <p>Content on this site is licensed under a <a href="http://creativecommons.org/licenses/by/4.0/">CC BY 4.0 License</a>. | Theme: <a href="https://github.com/peterdesmet/petridish">Petridish</a></p>

      
    </div>
  </div>
</footer>

<script src="/assets/theme/js/jquery.min.js"></script>
<script src="/assets/theme/js/popper.min.js"></script>
<script src="/assets/theme/js/bootstrap.min.js"></script>
<script src="/assets/theme/js/anchor.min.js"></script>

<script>
  // Show anchors when hovering over headings, using anchor.js
  (function () {
    const headings = ".content h2[id], .content h3[id], .content h4[id], .content h5[id], .content h6[id]";
    anchors.options = { placement: "right" };
    anchors.add(headings);
  })();
</script>



       
  </body>
</html>
